run/test/specs/group_disappearing_messages_image.spec.ts

```diff
@@ -13,6 +12,8 @@ async function disappearingImageMessageGroup(platform: SupportedPlatformsType) {
   const { device1, device2, device3 } = await openAppThreeDevices(platform);
   const testMessage = 'Testing disappearing messages for images';
   const testGroupName = 'Test group';
+  const time = DISAPPEARING_TIMES.THIRTY_SECONDS;
```

Comment:
Seems this happens in a few places so yeah, all should be outside or inside the function for each of the tests.

Reply:
Have updated them all to be outside